cvmfs/cvmfs_config

```diff
@@ -245,6 +312,12 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check that CVMFS_USER is set
+  if [ x"$CVMFS_USER" = x"" ]; then
```

Comment:
I agree, it doesn't hurt.  I'm just wondering if we will remember after some time why it is there.  Wouldn't the more accurate thing to test be if <ETCDIR>/cvmfs/default.conf exists?